src/backend/main.ts

```diff
@@ -629,7 +630,8 @@ ipcMain.handle('runWineCommand', async (e, args) => runWineCommand(args))
 ipcMain.handle('checkGameUpdates', async (): Promise<string[]> => {
   let oldGames: string[] = []
   const { autoUpdateGames } = GlobalConfig.get().getSettings()
-  for (const runner in libraryManagerMap) {
+  let runner: Runner = 'sideload'
+  for (runner in libraryManagerMap) {
```

Comment:
Was this change necessary?

Reply:
yes you can't cast a string to a runner, but you can cast a runner to a string

Reply:
nvm the cast worked this time idk